test/tests/pbs_Rrecord_resources_used.py

```diff
@@ -183,98 +188,121 @@ def common(self, is_nonrerunnable, restart_mom):
             self.momA.start(args=['-r'])
             self.momB.start(args=['-r'])
         else:
-            print "Mom is not running\n"
+            pass
```

Comment:
Correct me if I'm wrong, but isn't an "else: pass" the same thing as not having an else clause?